plugins/file/index.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
       },
       handler: async function(request, h) {
         const file = request.payload.file;
-        const fullPath = request.payload.fullPath;
+        const fileKey = request.payload.fullPath;
```

Comment:
This has to be `let fileKey` because it gets redefined later if `fileKey` is undefined.